.travis.yml

```diff
@@ -68,6 +71,31 @@ after_success:
                 docker push welder/web-cockpit
             fi
         fi
+after_failure:
+  - |
+        if [ "$TRAVIS_TEST_RESULT" ]; then
+            CASE_NUMBER=0
+            for SCREENSHOT in `ls ./failed-image`
```

Comment:
Aren't you missing a `;` at the end of this line, just before `do` ?

Also please move this into a separate script (called from make ci_after_failure) so we can make use of it in Jenkins if we decide to.